test/shared_setup.py

```diff
@@ -127,29 +127,35 @@ def test_action_target(self):
         """Test action_target function, mostly w.r.t. backward compatibility."""
         def fake_setup(*args, **kwargs):
             """Fake setup function to test action_target with."""
-            print 'args: ', args
-            print 'kwargs:', kwargs
+            print('args: %s' %''.join(args))
```

Comment:
@vsoch To be equivalent with the original, this probably needs to be:

```python
print('args: %s' % str(args))
```

See:

```
>>> args=()

>>> print('args: %s' % args)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: not enough arguments for format string

>>> print('args: %s' % ''.join(args))
args:

>>> print('args: %s' % str(args))
args: ()
>>> print 'args: ', args
args:  ()
```